Multilayer/readme.txt

```diff
@@ -0,0 +1 @@
+This folder includes all scripts for the Multilayer analysis . All instructions will be found here!
```

Comment:
I would recommend, at some point, writing in more detail a tutorial explaining these analyses/scripts.